fingertip/plugins/ssh.py

```diff
@@ -9,6 +9,9 @@ def main(m, no_unseal=False):
     with m.transient() as m:
         m.log.info(f'waiting for the SSH server to be up...')
         m.ssh.exec('true')
+        # terminate the ssh session not to leave any traces
+        m.ssh.invalidate()
```

Comment:
Does it bring any noticeable delay? Because I think proper cleanup in transient plugins might not be worth it for most purposes, we're dumping the machine anyway.

Reply:
I did not notice any noticeable delay as it just calls a simple `close()` of paramiko ssh session, properly cleaning the server processes in here.